### DISCUSSION TITLE
Advanced Foundry | IPFS Lesson | In Interactions File TokenURI Link to access PUG image  Attributes not working!

### DISCUSSION THREAD

**Discussion by vigneshbalu96** [Cyfrin/foundry-full-course-cu#3068] [Q&A] [answered]
:thinking:  Don't know why I am in Brave browser copied the link form the interactions file and tried but it throws error. This is as per the GitHub repo associated with the NFT course.
`ipfs://bafybeig37ioir76s7mg5oobetncojcm3c3hxasyd4rvid4jqhy4gkaheg4/?filename=0-PUG.json`

what would be the solution to work with above?

:point_down:  https link works fine.
`https://ipfs.io/ipfs/QmSsYRx3LpDAb1GZQm7zZ1AuHZjfbPkD6J7s9r41xu1mf8`

Thanks in Advance! :smile:

**Comment by EngrPips** [comment]
I think Brave has native support for IPFS, and you are supposed to be able to view your file with just this `ipfs://bafybeig37ioir76s7mg5oobetncojcm3c3hxasyd4rvid4jqhy4gkaheg4/?filename=0-PUG.json` but I have one other person complain about this same issue, I will advise you just use the `https` link that works for now.

**Reply by vigneshbalu96** [reply]
@EngrPips Just to know does :point_down: this link works fine for you and other fellow developers?

`ipfs://bafybeig37ioir76s7mg5oobetncojcm3c3hxasyd4rvid4jqhy4gkaheg4/?filename=0-PUG.json`

**Reply by EngrPips** [reply]
This `ipfs://QmfNN5LKRtjy4p9eaQbAZMiFzWhn6dtS3u8tf2WVNJX996` works for me If I have the content pinned to my local ipfs node and have my IPFS companion app running and also I have IPFS companion extension on my Chrome browser. The main gist is to ensure you have done everything I stated and just prepend your content `CID` with `ipfs://` and try fetching the content from your browser.

**Comment by vigneshbalu96** [comment, marked as answer]
Latest version of Brave browser don't support IPFS by default, as per my discussion with the Brave team here is their response :point_down: 
'
Essentially the number of people using the feature (less than 0.1%) did not justify the cost of maintaining it. There was a blog post made for this here with a bit more detail: [BLOG](https://brave.com/blog/ipfs-support/)
'

**Reply by EngrPips** [reply]
I see that makes sense.
